add openssh root login troubleshooting (#36109)
gravitational/teleport: docs/pages/server-access/troubleshooting-server.mdx

```

For more information about moderated sessions and session sharing, see 
[Moderated Sessions](../access-controls/guides/moderated-sessions.mdx).

> ## Unable to connect to Agentless OpenSSH server as root

Reviewer: ```suggestion
## Unable to connect to agentless OpenSSH server as root
```

> You should check sshd configuration in /etc/ssh/sshd_config for a setting like

Reviewer: ```suggestion
You should check your sshd configuration in `/etc/ssh/sshd_config` for a setting like
```

`PermitRootLogin no` or `PermitRootLogin forced-commands-only` - either of these
settings will prevent login as root. 

> If you wish to login as root to an OpenSSH server via Teleport, we recommend

Reviewer: ```suggestion
If you wish to log in as root to an OpenSSH server via Teleport, we recommend
```
"log in" is the verb phrase

Author: always gets me! thanks
